src/Uno.Core.Build/Uno - Backup.Core.Build.csproj

```diff
@@ -0,0 +1,85 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
```

Comment:
Nooooppppeee (the complete file has to be removed)

Reply:
Oops you're right

Reply:
Done!